ClinGen/clincoded
# Connects to #196 kl type number

Change Notes

* Schema updates
    * Any properties that hold a date/time now use ```date-time``` format.
    * Articles now have doi and pmcid properties, as well as its own dateTime property.
    * controlGroup removed
    * segregation object removed from its own schema and moved to the family schema.
    * "Yes"/"No" properties are now boolean instead of strings holding “yes” and “No.”
    * Fields that hold calculable numbers are now type integer instead of string.
    * Variants moved from family object to segregation object.
    * statistic schema removed.
    * Variants have their own dateTime property.
    * Test data modified to match schema changes

* UI Changes
    - dateTime now added to new articles.
    - Settinng new boolean and integer values in the method and demographic objects.

The UI is now compatible with the schema changes. Testing this requires a typical curation path through group creation:
* Create a new gene-disease record.
* Add a PubMed from the PubMed site.
* Click the new article and add group information.
* Save the group information.
* View the group, and it should show the same information you entered.
